crates/nargo_cli/tests/execution_success/eddsa/src/main.nr

```diff
@@ -0,0 +1,55 @@
+use dep::std::compat;
+use dep::std::ec::consts::te::baby_jubjub;
+use dep::std::hash;
+use dep::std::eddsa::eddsa_poseidon_verify;
+use dep::std;
+
+fn main(msg: pub Field, _priv_key_a: Field, _priv_key_b: Field) {
```

Comment:
Why was this added?

Reply:
Can you add this as a separate PR? I think it was taking a long time, but perhaps its managable now?